Tests/test_file_eps.py

```diff
@@ -220,7 +311,8 @@ def test_read_binary_preview():
         pass
 
 
-def test_readline(tmp_path):
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
```suggestion
```

Rather than filtering out all `DeprecationWarning`s (there could potentially be warnings in the future from elsewhere that we would want to know about), we can put a `with pytest.warns(DeprecationWarning):` context manager around the specific `PSFile` call.